website/src/controllers/api-v3/tasks.js

```diff
@@ -102,10 +102,11 @@ api.createChallengeTasks = {
     if (challenge.leader !== user._id) throw new NotAuthorized(res.t('onlyChalLeaderEditTasks'));
 
     let tasks = await _createTasks(req, res, user, challenge);
-    res.respond(201, tasks.length === 1 ? tasks[0] : tasks);
 
     // If adding tasks to a challenge -> sync users
-    if (challenge) challenge.addTasks(tasks); // TODO catch/log
+    if (challenge) await challenge.addTasks(tasks); // TODO catch/log
```

Comment:
this has to be placed after res.respond. In big challenges it may take a while so we want this to run in background